Dockerfile.template

```diff
@@ -61,7 +61,7 @@ ENV LC_ALL ${DEFAULT_LOCALE}
 #
 
 # Set up environment variables for Silverpeas
-ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
+ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-TARGET_ARCH
```

Comment:
Having a separate `Dockerfile` per architecture just for changing one environment variable is not going to be acceptable for the official images.

See https://github.com/docker-library/openjdk/blob/ecb81629f5d5f226491e9f3e0a5be11662f72870/11/jdk/Dockerfile#L28-L40 for the approach we took over in the `openjdk` official images which might be acceptable to the maintainers here?